mpas_analysis/config.default

```diff
@@ -171,7 +171,10 @@ htmlSubdirectory = html
 # option:
 #    mpas_analysis config.analysis --generate \
 #         only_ocean,no_timeSeries,timeSeriesSST
-generate = ['all_publicObs', 'no_icebergs']
+generate = ['all_publicObs', 'no_icebergs', 'no_climatologyMapAntarcticMelt',
+            'no_timeSeriesAntarcticMelt', 'no_timeSeriesOceanRegions',
+            'no_climatologyMapSose', 'no_climatologyMapSchmidtko',
+            'no_soseTransects']
```

Comment:
@milenaveneziani, this is the list of tasks I came up with to disable by default.  Anything else you see?  Any of these you wouldn't disable?

Reply:
@xylar, I normally use `no_landIceCavities`, but that is probably equivalent to `no_climatologyMapAntarcticMelt`  and `no_timeSeriesAntarcticMelt`?
Also, what about disabling `oceanRegionalProfiles`?

Reply:
>  I normally use no_landIceCavities, but that is probably equivalent to no_climatologyMapAntarcticMelt and no_timeSeriesAntarcticMelt?

I'm not sure but I'd rather explicitly disable the tasks instead of using the tags to make things a little more obvious.  I could do this with the iceberg task, too.

> Also, what about disabling oceanRegionalProfiles?

I can do this.  Is this just because it's expensive?  It's not something specific to polar regions.  It was requested by @pwolfram awhile back but I don't know that it's proven particularly useful.  Too bad, because it was a ton of work...

Reply:
Regarding `'no_icebergs'`, I do think the simplicity compared with `'no_climatologyMapIcebergConcNH', 'no_climatologyMapIcebergConcSH'` is justified.

Reply:
oh no, I'm not saying it is not useful! (in fact,  I was thinking to eventually find a way to add any available observations to it).
I've had to disable it because it's slow when it has to compute region masks on the fly.

Reply:
I see.  That's a good reason.

Regarding adding obs, I think they could be pulled from WOA18.  Similar problem where generating masks on the fly is slow, but if you given it enough dask threads, it should be fast.  (At least, it should be equivalent to the standalone script for creating masks.)

Reply:
Anyway `oceanRegionalProfiles` is now disabled by default.